src/Renderer/ThreeExtended/OBB.js

```diff
@@ -7,17 +7,21 @@
 
 import * as THREE from 'three';
 
-function OBB(min, max, lookAt, translate) {
+function OBB(min, max, lookAt, translate, up) {
     THREE.Object3D.call(this);
     this.box3D = new THREE.Box3(min, max);
 
     this.natBox = this.box3D.clone();
 
     this.quaInv = this.quaternion.clone().inverse();
 
+    if (up) {
+        this.up.set(up.x, up.y, up.z);
```

Comment:
The lookAt function needs to know which way is up. Up by default is (0,1,0). Since we changed the "main axis" from y to z, we also need to change the up vector when initialising our OBB.